### ISSUE TITLE
(develop) `cargo pgx new` not generates `pgx-linker-script.sh` in it's template

### ISSUE THREAD

**Issue by comdiv** [zombodb/pgx#202]
So after `cargo pgx new my_ext` 
Building falling with linker `/home/comdiv/code/hw/./.cargo/pgx-linker-script.sh` not found

**Comment by comdiv** [comment]
Have added manually from pgx repository itself:
```
#! /usr/bin/env bash
# Auto-generated by pgx. You may edit this, or delete it to have a new one created.

if [[ $CARGO_BIN_NAME == "sql-generator" ]]; then
    gcc -Wl,-undefined,dynamic_lookup,-dynamic-list=$CARGO_MANIFEST_DIR/.cargo/pgx-dynamic-list.txt $@
else
    gcc -Wl,-undefined,dynamic_lookup $@
fi
```

**Comment by Hoverbear** [comment]
Thanks for this! We spoke on Discord and you have it sorted. :)

**Comment by Hoverbear** [comment]
This is fixed in #197 which is merged into `develop`. :)
